lib/move_loader.rb

```diff
@@ -0,0 +1,19 @@
+require 'web_human_player'
+
+class MoveLoader
+  def preload(player, move)
+    if web_human?(player) && valid_move?(move)
```

Comment:
There is not always a move selected from the gui (when the game type is Ai vs Human, or for the other game types, when the human goes first, the grid is presented before a move is chosen). This means, I have to check for nil somewhere (as I don't want to do to_i on a nil, and get the value 0, and plant a move at position 0 inappropriately).

I put it here, rather than higher up, as this object deals with setting the valid move on the first player, if appropriate.